examples/web-ui/material/pages/Design/designHome.smash

```diff
@@ -0,0 +1,17 @@
+* Given I am on design page
+    Navigate to design page
+        On design page
+
+* Navigate to design page
+    Navigate to 'https://material.io/design'
+        On design page
+
+* On design page
+    setDesignPage +? {
+        props({
+            'designH1': `h1, "Design"`
```

Comment:
Great EF! You may want to call it something other than `designH1` though. Maybe `design header`?